paasta_tools/paasta_cli/cmds/mark_for_deployment.py

```diff
@@ -109,4 +113,21 @@ def paasta_mark_for_deployment(args):
             cluster=cluster,
             instance=instance,
         )
+    return returncode
+
+
+def paasta_mark_for_deployment(args):
+    """Wrapping mark_for_deployment"""
+    cluster, instance = args.clusterinstance.split('.')
+    service = args.service
+    if service and service.startswith('services-'):
```

Comment:
ah ok, makes sense. could you add a comment to explain that? In the long term it'd be good to move this logic out of paasta and into jenkins (we could just sed the $upstream_job_name that Jenkins passes as an arg)